adsingestp/parsers/jats.py

```diff
@@ -896,6 +897,11 @@ def _parse_page(self):
                     self.article_meta.find("counts").find("page-count").get("count", "")
                 )
 
+    def _parse_fulltext(self):
+        if self.body is not None:
+            body_text = str(self.body.extract())
```

Comment:
In the current fulltext pipeline, we do a lot of manipulation on the extracted body (see here for an example: https://github.com/adsabs/ADSfulltext/blob/master/adsft/extraction.py#L710). Will we be passing this through the fulltext pipeline to do that manipulation? What will this parsed fulltext be used for at the moment?

Eventually, I'd like to rethink how the fulltext pipeline operates, in conjunction with these parsers, but that's a bigger task.